### PR TITLE
Add issue template for general improvements

### PR DESCRIPTION
**Pull request by kerber0x**
## Description and Motivation

This PR adds an issue template for general improvements as it was missing. 

## Related Issues
#43 

---
## Checklist:

- [x] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-core/blob/main/CONTRIBUTING.md).
- [x] My pull request has a sound title and description (not something vague like `Update index.md`)
- [ ] All existing and new tests are passing.
- [x] I updated/added relevant documentation.
- [ ] The code is formatted properly `cargo fmt --all --`.
- [ ] Clippy doesn't report any issues `cargo clippy -- -D warnings`.
- [ ] I have regenerated the schemas if needed `cargo schema`.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2_improvement.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+name: Improvement proposal
+about: Improvements to the contracts, the website, the copy... Anything that is working well but could be enhanced.
```
Comment (kaimen-sano) [inline .github/ISSUE_TEMPLATE/2_improvement.md:3]: What is `copy...`?

File: .github/ISSUE_TEMPLATE/2_improvement.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+     or consider asking a question on Discord's smart contract channel:
+      * https://whitewhale.money/
+      * https://white-whale-defi-platform.github.io/docs/
+      * https://discordapp.com/channels/908044702794801233/987301947440767006
+
+     If you have found a bug or if our documentation doesn't have an answer
+     to what you're looking for, then fill out the template below.
+-->
+
+If you want to report a security issue, please follow our security policy: `https://github.com/White-Whale-Defi-Platform/migaloo-core/blob/main/SECURITY.md`
```
Comment (kaimen-sano) [inline .github/ISSUE_TEMPLATE/2_improvement.md:22]: Should this be included in the comments above?

**Review by kaimen-sano — APPROVED** [review]
What distinguishment do we have between a feature request and an improvement? I worry that users may get a bit confused between the two and use the wrong one. Overall changes look fine though.

File: .github/ISSUE_TEMPLATE/2_improvement.md (hunk 1 above)
Comment (0xFable) [reply .github/ISSUE_TEMPLATE/2_improvement.md:3]: do you mean the tests?, what is the copy 🧐

**Review by 0xFable — APPROVED** [review]
LGTM other than Kaimens comments, great work ser

File: .github/ISSUE_TEMPLATE/2_improvement.md (hunk 1 above)
Comment (kerber0x) [reply .github/ISSUE_TEMPLATE/2_improvement.md:3]: Copy as in copywriting, so changes in the text on the website for instance.

File: .github/ISSUE_TEMPLATE/2_improvement.md (hunk 2 above)
Comment (kerber0x) [reply .github/ISSUE_TEMPLATE/2_improvement.md:22]: I think it's better to have it there so they don't miss it, in the comment they maybe miss it by mistake and report a security issue publicly

**Comment by kerber0x** [comment]
@kaimen-sano the difference of this one with feature request is that feature request is something we don't have, while an improvement is something that exists but could be enhanced, does it make sense? 

Maybe we could somehow have a single template for both feature requests / improvements so people don't get confused? What do you think @0xFable @kaimen-sano

File: .github/ISSUE_TEMPLATE/2_improvement.md (hunk 1 above)
Comment (kaimen-sano) [reply .github/ISSUE_TEMPLATE/2_improvement.md:3]: Which website exactly? Do we have one for the migaloo core repo?

File: .github/ISSUE_TEMPLATE/2_improvement.md (hunk 2 above)
Comment (kaimen-sano) [reply .github/ISSUE_TEMPLATE/2_improvement.md:22]: Sounds reasonable

**Comment by kaimen-sano** [comment]
> The difference of this one with feature request is that feature request is something we don't have, while an improvement is something that exists but could be enhanced, does it make sense?
> 
> Maybe we could somehow have a single template for both feature requests / improvements so people don't get confused? What do you think

This difference seems reasonable to me, perhaps we should include this information more explicitly in the summary when users are selecting? Something like `Upgrade to an existing feature` or `Propose a new feature`?

File: .github/ISSUE_TEMPLATE/2_improvement.md (hunk 1 above)
Comment (kerber0x) [reply .github/ISSUE_TEMPLATE/2_improvement.md:3]: Not a specific one, can be the porject or the app website, it's more about general improvements across the project.

**Pull request merged**